materials/01-what-is-vc.qmd

```diff
@@ -2,12 +2,63 @@
 title: "1. What is Version Control"
 ---
 
-## here's some code to copy
+## 1.1 What is Version Control?
 
-```{r}
-#| echo: true
-#| code-copy: true
-rnorm(10)
+Version control, also known as source control, is the practice of tracking and managing changes to software code. Version control systems are software tools that help software teams manage changes to source code over time.
```

Comment:
I might mention you don't have to specifically be a software team (as most BC stats'ers are not). Just anyone with any type of code/documents that you want under VC. Most useful for code for sure but has other uses.